modules/persistent-storage-csi-migration-vsphere.adoc

```diff
@@ -6,82 +6,53 @@
 [id="persistent-storage-csi-migration-sc-vsphere_{context}"]
 = vSphere automatic migration
 
-== New installations of {product-title}
-For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
-
-[id="upgrading-openshift_{context}"]
-== Upgrading {product-title}
-When upgrading from {product-title} 4.12, or earlier, to 4.13, automatic CSI migration for vSphere only occurs if you opt in.
-
 [IMPORTANT]
 ====
-Carefully review the following consequences before opting in to migration:
-
-* link:https://access.redhat.com/node/7011683[There are known issues that can occur that can cause the migration to fail].
+Migration includes significant consequences:
 
 * Enabling migration cannot be undone.
 
-* Migration can take awhile to complete depending on how many nodes are on the cluster.
-
-* Migration is a disruptive process. The Machine Config Operator (MCO) has to update kubelet, which means degrading each node as it rolls out the new MachineConfig to each machine.
+* Migration can take a while to complete depending on how many nodes are on the cluster.
 ====
 
-=== Using the web console to opt in to automatic CSI migration
-
-==== Prerequisites
-
-* Access to the {product-title} web console.
-
-* Access to the cluster with cluster-admin privileges.
-
-===== Procedure
-
-To opt in to automatic CSI migration for vSphere:
-
-. Log in to the web console.
-
-. Click *Administration* -> *CustomResourceDefinitions*.
-
-. On the *CustomResourceDefinitions* page, type "Storage" to find the `Storage` custom resource (CR).
-
-. Click the *Storage* CR.
+== New installations of {product-title}
+For new installations of {product-title} 4.13, or later, automatic migration is enabled by default.
 
-. On the *storages.operator.openshift.io* page, click the *Instances* tab.
+[id="updating-openshift_from_4.13_{context}"]
+== Updating from {product-title} 4.13 to 4.14
+If you are using vSphere in-tree persistent volumes (PVs) and want to update from {product-title} 4.13 to 4.14, update vSphere vCenter and ESXI host to 7.0 Update 3L or 8.0 Update 2, otherwise the {product-title} update is blocked. After updating vSphere, your {product-title} update can occur and automatic migration is enabled by default.
```

Comment:
Unless CSI migration is already enabled.
cc @gnufied

Reply:
Yeah - so something on the lines of "If you are using vSphere in-tree PVs and have not opted-in to CSI migration in 4.13, update from 4.13 to 4.14 will be blocked......."

Reply:
@gnufied @gcharot agreed not add more complexity to this statement in the